### PR TITLE
Lps 152440 7 11

### PR DESCRIPTION
**Pull request by slnn**
@tinatian No logic chagne, just update the bnd to resolve the conflict.

@catarineaguiar please scan the test results in this pull, thanks!

Lily

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@slnn no regressions found.

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateCollectionDisplayConfigMVCActionCommand.java [hunk 1]
```diff
@@ -107,16 +107,12 @@ protected void doProcessAction(
 				FragmentEntryProcessorConstants.
 					KEY_FREEMARKER_FRAGMENT_ENTRY_PROCESSOR);
 
-			if ((configuration == null) || !JSONUtil.isValid(configuration)) {
-				continue;
-			}
+			JSONObject configurationJSONObject = JSONUtil.createJSONObject(
```
Comment (tinatian) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateCollectionDisplayConfigMVCActionCommand.java:110]: @slnn You did not check (configuration == null), there may be an NPE after your change.

**Comment by tinatian** [comment]
@slnn Please see my comment and resend.

Thanks.
Tina

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateCollectionDisplayConfigMVCActionCommand.java (hunk 1 above)
Comment (dantewang) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateCollectionDisplayConfigMVCActionCommand.java:110]: @tinatian Actually the null protection is not needed. In the new PR the null check is removed in a separate commit to make it clear: https://github.com/liferay-core-infra/liferay-portal/pull/1101/commits/8f41ce2502d5ad2e90f707564951294eb6f8586d

**Pull request closed without merging**